w04/test/gad/simplehash/UnitTest.java

```diff
@@ -0,0 +1,653 @@
+package gad.dynamicarray;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+
+// import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+//
+// HALT STOPP! LESEN!
+//
+// TESTS SIND GE'SPEEDRUN'T UND OHNE QUALITÄT VERFASST
+// DISABLED = ICH [=VERFASSER] PASSE ARTEMIS TESTS [=VIELLEICHT] ABER TESTS NICHT [=SICHER]
```

Comment:
You selected
> - [x] I confirm that my tests, if any, are correct
so this sentence should be obsolete (?)

Reply:
Dead code indeed